test/state/block.hpp

```diff
@@ -51,11 +51,9 @@ struct BlockInfo
 
     /// The "excess blob gas" parameter from EIP-4844
     /// for computing the blob gas price in the current block.
-    uint64_t excess_blob_gas = 0;
+    std::optional<uint64_t> excess_blob_gas;
 
-    /// The blob gas price parameter from EIP-4844.
-    /// This value is not stored in block headers directly but computed from excess_blob_gas.
-    intx::uint256 blob_base_fee = 0;
+    std::optional<uint64_t> blob_gas_used;
```

Comment:
Add doc comment.
Also place before `excess_blob_gas` to keep the spec order from https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4844.md#header-extension.